packages/slice-machine/pages/changes.tsx

```diff
@@ -32,6 +35,27 @@ const Changes: React.FunctionComponent = () => {
     authStatus,
     isOnline,
   } = useUnSyncChanges();
+
+  type ChangedSlice = { status: ChangesStatus; c: ComponentUI };
```

Comment:
Can we define these somewhere else? And rename `c` (to `slice` and `customType` below, unless it makes sense to have a common type in which case probably `component`)